docs/reference/pyproject-settings.rst

```diff
@@ -31,3 +31,19 @@ This page lists the configuration settings supported by ``meson-python`` in the
 
    * - ``tool.meson-python.args.install``
      - Extra arguments to be passed to the ``meson install`` command.
+
+Usage example:
+
+.. code-block:: toml
+
+   [tool.meson-python.args]
+   setup = ['-Dwarning_level=2', '-Db_pie=true']
+   dist = ['--include-subprojects']
+   compile = ['-j4']
+   install = ['--quiet']
+
+.. note::
+
+   At the moment it is necessary to specify all four arguments (``dist``,
+   ``setup```, ``compile`` and ``install``). This limitation will go away in a
+   future version.
```

Comment:
This was fixed in #304. In general, I think it is a good idea to link to a bug report any time we say "in a future version" so that the user can check whether that means in practice.

Reply:
Ah, great!

>  In general, I think it is a good idea to link to a bug report any time we say "in a future version" so that the user can check whether that means in practice.

Agreed. I looked for one but couldn't find it. I think it was discussed in other PR review.

Reply:
Removed this note, so resolving the comment.